mimic-iv/concepts/concept_map/d_labitems_to_loinc.csv

```diff
@@ -1,1631 +1,1624 @@
-52023,Assist/Control,Blood,Blood Gas,,,,Measurement,LOINC,Lab Test,S,,,,MIMIC-IV v2.0,,,,,,Not a lab test,22386
-52024,"Creatinine, Whole Blood",Blood,Blood Gas,mg/dL,3051825,Creatinine [Mass/volume] in Blood,Measurement,LOINC,Lab Test,S,38483-4,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,4123
-52025,Delete,Blood,Blood Gas,,,,Measurement,LOINC,Lab Test,S,,,,MIMIC-IV v2.0,,,,,,Not a lab test,0
-52026,Estimated GFR (MDRD equation),Blood,Blood Gas,,,,Measurement,LOINC,Lab Test,S,76633-7,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,3538
-52027,"Glucose, Whole Blood",Blood,Blood Gas,,3000483,Glucose [Mass/volume] in Blood,Measurement,LOINC,Lab Test,S,2339-0,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,0
-52028,Hematocrit,Blood,Blood Gas,,3023230,Hematocrit [Volume Fraction] of Arterial blood,Measurement,LOINC,Lab Test,S,32354-3,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,0.5,,0
-52029,% Ionized Calcium,Blood,Blood Gas,,3032710,Calcium.ionized/Calcium.total corrected for albumin in Blood,Measurement,LOINC,Lab Test,S,49936-8,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,0
-52030,Lithium,Blood,Blood Gas,,3001823,Lithium [Moles/volume] in Blood,Measurement,LOINC,Lab Test,S,25461-5,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,0
-52031,Osmolality,Blood,Blood Gas,,3008295,Osmolality of Serum or Plasma,Measurement,LOINC,Lab Test,S,2692-2,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,0
-52032,P50 of Hemoglobin,Blood,Blood Gas,mm Hg,3015968,Oxygen [Partial pressure] saturation adjusted to 0.5 in Arterial blood,Measurement,LOINC,Lab Test,S,19214-6,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,,1
-52033,Specimen Type,Blood,Blood Gas,,40769406,Specimen type,Measurement,LOINC,Lab Test,S,66746-9,,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,,Not a lab test,664819
-52034,Total Calcium,Blood,Blood Gas,,3032503,Calcium [Mass/volume] in Blood,Measurement,LOINC,Lab Test,S,49765-1,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,0.1,,0
-52035,Total Calcium,Blood,Blood Gas,,3032503,Calcium [Mass/volume] in Blood,Measurement,LOINC,Lab Test,S,49765-1,0000-0001-8822-1884,0000-0002-9348-9284,MIMIC-IV v2.0,2.71,https://loinc.org/relma/,v4.7,3/1/2022,0.1,Duplicate of itemid 52034,0
-52036,Voided Specimen,Blood,Blood Gas,,,,Measurement,LOINC,Lab Test,S,,,,MIMIC-IV v2.0,,,,,,Not a lab test,6397
-52037,WB tCO2,Blood,Blood Gas,,3014094,"Carbon dioxide, total [Moles/volume] in Blood",Measurement,LOINC,Lab Test,S,20565-8,0000-0001-882
```

Comment:
It depends. For example, itemid 52036 is for the concept "Voided Specimen" which is not a lab test. 
Other concepts have ambiguous labels that don't make any sense.
Do you think I should use "there is no exact mapping" for all unmappable concepts regardless of the reason?

Reply:
This is an interesting question for the sssom issue tracker https://github.com/mapping-commons/sssom/issues because what we usually do in this case is _not include the subject entity_ at all in the mapping file. If there is no mapping there is no mapping. However, I can see how your desire to have an entry for all subject entities in MIMIC makes sense. In this case, we usually define the predicate_id (e.g skos:exactMatch) and then add a placeholder, `sssom:NoTermFound`, see https://github.com/mapping-commons/sssom/issues/28 for a discussion on the subject. This will allow us to also later say: there is no broad/narrow match easier. However, for your case, this solution is not perfect, because you really want to say something like "out of scope", but that's not currently possible to express, so maybe make an issue?

Reply:
Thanks @matentzn, I think you're right that `sssom:NoTermFound` isn't quite what we need. @a-chahin give me a shout if you'd like to discuss this before raising the issue. Something like an `OutOfScope` tag sounds good to me!

Reply:
I have just raised an issue at https://github.com/mapping-commons/sssom/issues/245